### PR TITLE
Fix Content managment scripts

### PR DESCRIPTION
**Pull request by adi88d**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
- Update **Configuration Setup** playbook - install custom packs **after** installing the marketplace packs to avoid the error:
`could not install content pack\",\"error\":\"create / update /delete operation is already in progress `
- **CustomPackInstaller** script - Added the *instance_name* parameter to specify `Demisto REST API` instance to use.
- **MarketplacePackInstaller** script - Added the *instance_name* parameter to specify `Demisto REST API` instance to use and fix the code to be more stable.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ContentManagement/Scripts/CustomPackInstaller/CustomPackInstaller.yml [hunk 1]
```diff
@@ -19,6 +19,12 @@ args:
   predefined:
   - "true"
   - "false"
+- default: true
```
Comment (yaakovi) [inline Packs/ContentManagement/Scripts/CustomPackInstaller/CustomPackInstaller.yml:22]: no need to add an argument, use the builtin `using` argument

File: Packs/ContentManagement/Scripts/MarketplacePackInstaller/MarketplacePackInstaller.yml [hunk 2]
```diff
@@ -18,6 +18,12 @@ args:
   name: pack_version_key
   required: true
   secret: false
+- default: true
```
Comment (yaakovi) [inline Packs/ContentManagement/Scripts/MarketplacePackInstaller/MarketplacePackInstaller.yml:21]: same

File: Packs/ContentManagement/Playbooks/playbook-Configuration_Setup.yml [hunk 3]
```diff
@@ -373,35 +375,37 @@ tasks:
...
+      - '6'
     scriptarguments:
       pack_id_key:
         simple: packid
       pack_version_key:
         simple: packversion
       packs_data:
         simple: ${ConfigurationSetup.MarketplacePacks}
+      instance_name:
+        simple: Demisto REST API Gold
```
Comment (yaakovi) [inline Packs/ContentManagement/Playbooks/playbook-Configuration_Setup.yml:402]: same

**Review by yaakovi — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@yaakovi Doc review completed.

**Comment by ShirleyDenkberg** [comment]
@adi88d Just want to confirm that the REST API is called Demisto REST API and not Cortex XSOAR REST API.

**Comment by adi88d** [comment]
@ShirleyDenkberg yes, this is the name of the integration - `Demisto REST API`

**Pull request merged**